### DISCUSSION TITLE
Proposal for Redesigning the IDE Logo

### DISCUSSION THREAD

**Discussion by AlainRichell** [eclipse-platform/eclipse.platform.releng.aggregator#2432] [Ideas]
Dear developers,

I hope this message finds you well.

I would like to propose a redesign of the Eclipse IDE logo, with the objective of modernizing its overall look while maintaining the strong foundation that has been built around eclipse current brand identity. The idea behind this redesign is to align with the modern trends we see across major enterprises, adopting a minimalistic approach that feels fresh and contemporary but still recognizable.

The new design preserves the existing color palette, ensuring that the essence and strength of the brand remain intact. However, I’ve made some adjustments to the form and style, giving it a sleeker and more refined appearance, which I believe will resonate well with both current users and potential new audiences.

I have already created a draft of the proposed logo and will be sending it shortly for your review. I would be happy to discuss the design further and make any adjustments based on your feedback.

--------------------------------------------------------------------------------------------------------------------------------------------

![Eclipse Design LT_Mesa de trabajo 1](https://github.com/user-attachments/assets/1944501e-6a3c-4845-bc45-f2da044da76f)

--------------------------------------------------------------------------------------------------------------------------------------------

Thank you for considering this proposal. I’m confident that this small but impactful change can help modernize the product’s image and enhance its appeal.

Best regards,
Alain Richell - Java Developer

**Comment by tfroment** [comment]
Hi @AlainRichell, as Eclipse IDE Program Manager, I am curious to get the feedback from the community on your idea & proposal, and I believe we need to share it with the Eclipse IDE WG Members and the Eclipse Foundation  because the impact of such a change goes far beyond the look & feel; it involves accompanying it with a project, whether it's updating all the tools, communication materials, websites, goodies, etc...

My personal thinking (without involving yet the Foundation or the Working Group): what is certain is that in the continuation of the ongoing modernization initiatives that will be presented at EclipseCon (I am thinking, for example, of Initiative31: https://www.ocxconf.org/event/778b82cc-6834-48a4-a58e-f883c5a7b8c9/websitePage:26bc9d91-446a-49e1-9f08-8fcc4680073e?session=a378e7e9-420c-482f-8944-17a7cf88a7b6&shareLink=true), this kind of idea could be an opportunity for a communication plan in 2025, and it's great that community members are proposing such things. That's why I'm therefore curious to know the community's opinion on this, and I suggest we stay connected on this topic: will you be attending EclipseCon? It would be a good opportunity to discuss it in person.

**Comment by opcoach** [comment]
I like it. Modern and dynamic.  When I saw it the first time I believed it was related to the Euro €  but Euro symbol has 2 lines inside. Then if we analyze it more, we can see a 'E' and a 'C' which are the 2 first letters of Eclipse ...

**Reply by tfroment** [reply]
> I like it. Modern and dynamic. When I saw it the first time I believed it was related to the Euro € but Euro symbol has 2 lines inside. Then if we analyze it more, we can see a 'E' and a 'C' which are the 2 first letters of Eclipse ...

I agree it has something in common with:
![image](https://github.com/user-attachments/assets/7292a653-b38c-4971-ac87-1cdae7111d11)

**Comment by laeubi** [comment]
Just for reference if anyone wonder how it currently looks like https://www.eclipse.org/org/artwork/#eclipse-ide-logos

**Comment by iloveeclipse** [comment]
The idea of updating the logo is fine, however the proposed logo doesn't look good IMO. 
The "e" looks to much like € sign and the background graphics don't make much sense for me too.

I think **if** there is a need for a new / updated logo, we should have much more and different proposals.

**Reply by mickaelistria** [reply]
I support that before starting moving forward, stakeholders/contributors should evaluate "if there is a need for a new / updated logo" first. Such changes in branding a usually exhausting for the community, leading to endless debates and some potential extra stress, and are relatively expensive (need to change logos in code, on website, in doc, on marketplace...). So unless there is an issue with the current one or unless there is a clear enough indication that the effort is worth the pain, I would recommend to not bother with it.
I never heard someone leaving from Eclipse IDE to competition because of its logo. It's IMO a non-problem.

**Comment by pcdavid** [comment]
Personally I'm not against a refresh, but the proposal is too different from the existing one IMO. I'm not sure throwing away 20+ years of brand recognition is a good idea.

**Comment by jukzi** [comment]
I personally associate Eclipse mostly by the logo used in the taskbar /titlebar
![image](https://github.com/user-attachments/assets/4991af04-b453-4989-a184-6f17a7481f02)
![image](https://github.com/user-attachments/assets/2d9e68f5-0a80-4862-ae0c-ef682230921f)

And that is already pretty and simplistic enough i think.
The only thing that could be even more simplistic: The orange is barely visible and for me eclipse is basiscally blue+white. The orange part could be dropped at all.

**Reply by BeckerWdf** [reply]
> The orange is barely visible and for me eclipse is basiscally blue+white

But this is due to your not-so-high-resolution display. On mine a can clearly see it:
<img width="130" alt="image" src="https://github.com/user-attachments/assets/9d33a6cd-e2c0-4b1c-89d7-1fae96f584fd">

**Reply by HannesWell** [reply]
Personally I also like the idea to drop the orange color for the [solar corona](https://en.wikipedia.org/wiki/Solar_eclipse#Total_eclipse) and making the logo just plain blue and white. I would also make the body more dark-blue and less purple and if the 3D-effect is removed it's also full flat style.

This would be similar to the logo shown on top at https://eclipseide.org/ (but not full white):
![grafik](https://github.com/user-attachments/assets/3592f3a4-4035-44bc-b878-0561bc9fcdc2)

This would not be too disruptive and would still be recognizable as Eclipse logo we are all used to, but should still be a modernization.

Maybe I've seen this variant already somewhere, but I don't recall where exactly.

**Reply by jensli** [reply]
> This would not be too disruptive and would still be recognizable as Eclipse logo we are all used to, but should still be a modernization.

I think this sounds like a good idea. I like this version of the logo, the circle with stripes. I think it is distinctive, pretty and simple.

**Reply by laeubi** [reply]
What adds another important aspect, such al logo has to be clearly visible in large to small scales, that sadly something that is often forgotten.

This is how the proposal looks at 32x32
![grafik](https://github.com/user-attachments/assets/09cf6331-5c63-4dd1-8e4e-842bc9ed6f4a)

and 16x16 (the default size for eclipse icons at the moment)
![grafik](https://github.com/user-attachments/assets/4aa75c86-9e97-43aa-b674-21d666adeaeb)

**Reply by jukzi** [reply]
With simplistic three lines one could also make the middle line shorter to allude an Eclipse "E"
![image](https://github.com/user-attachments/assets/719bd187-55b4-44d3-bdd3-7fb6640c29f2) ![image](https://github.com/user-attachments/assets/634e2b9e-e5c5-42ab-8903-84ffbbd8c4d4)

**Comment by BeckerWdf** [comment]
In the proposal you no longer can see the "solar eclipse". I mean something like this:

![image](https://github.com/user-attachments/assets/03315fd7-b06a-4ba0-8b7e-d2b7b4f31c9d)

And I also see an € sign

**Reply by garydgregory** [reply]
Woa, that one is great! The proposal at the start of this ticket is... ugh, not great IMO, bland, boring. The current logo in 4.33.0 is beautiful 😍

**Reply by pcdavid** [reply]
> The current logo in 4.33.0 is beautiful 

Are you referring to [the splash screen](https://raw.githubusercontent.com/eclipse-platform/eclipse.platform/master/platform/org.eclipse.platform/splash.png)? This is not the same thing as the logo that is discussed here.

**Comment by merks** [comment]
If you look at it just the right way, you can see a monkey wrench with a white background behind it and then there's no `e` at all.  😜 

Maybe it helps to look at it from a different angle:

![image](https://github.com/user-attachments/assets/c2e3d297-a65d-4726-b2a0-e615d8e9875f)

**Reply by garydgregory** [reply]
Boring IMO.

**Comment by rgrunber** [comment]
At a minimum we should review https://bugs.eclipse.org/bugs/show_bug.cgi?id=300271 , https://ianskerrett.wordpress.com/2010/02/24/call-for-community-feedback-on-an-updated-eclipse-logo/ , https://ianskerrett.wordpress.com/2010/03/09/results-of-round-1-logo-feedback/ to get a better idea of why it was done, why it seemed to fail in 2010, and was retried in 2013 ? There needs to be a strong argument to do this because there's probably a lot of resources that will go into having legal vet the possible options, updating the old logo references, and marketing.

With that said, it reminds me a bit of https://en.wikipedia.org/wiki/Rebel_Alliance (rotated). The colour palette is nice but the logo doesn't really convey "lightweight/minimal" to me. Maybe something like https://en.wikipedia.org/wiki/E_(mathematical_constant) , would be a bit better for that, but then that might be getting too close to the IE logo.

**Comment by AlainRichell** [comment]
Thank you all for the different points of view and kind responses. Just
take the proposal as it is, a proposal. Unlike some of you believe, I like
the old logo too but everything is up to change at some point, or maybe
not, like BIC (which remains good), but if you think about it, generations
change, and there are new ways of aproaching how the newcomers engange with
the tools enterprises provide them, that includes, not only the logo but
the entire workflow of the program. I love eclipse, and thats why this idea
came up, to bring the IDE closer to the new programmers standards. Maybe
this design I made wont be the one that make the change but you can get an
idea from it, and I will make other designas too as I've already made.
Anyways, thanks for your time to all of you, I'll stay tuned to anything
else you would like to share. :)

**Comment by AlainRichell** [comment]
Lastly but not less important, the logo isn't an € hahaha. It is meant to
be an "E" quarter moon shaped in front of a rising eclipse, maybe it just
needs a better design.

**Comment by carstenartur** [comment]
Is it possible to make use of transparency for the Logo and the splashscreen?

**Reply by AlainRichell** [reply]
of course it is, though it wasn't mean to be a splashscreen that way
![Eclipse Design LT](https://github.com/user-attachments/assets/d60f6074-9ed0-4d2d-9cdf-8ea71f277ac2)
![Eclipse I](https://github.com/user-attachments/assets/22ef9d4f-7321-49b3-b9dd-214a2bd6c32f)

**Reply by carstenartur** [reply]
So something like this is possible?

https://stackoverflow.com/questions/22979232/how-to-design-a-cool-semi-transparent-splash-screen

**Reply by AlainRichell** [reply]
if it was possible sometime ago why it wouldn't be now?

**Comment by gireeshpunathil** [comment]
I love eclipse, and I love change. hence I like this proposal! - for a drastic change towards simplicity.
